src/librustc_typeck/check/compare_method.rs

```diff
@@ -1164,7 +1164,7 @@ fn compare_type_predicate_entailment(
 fn assoc_item_kind_str(impl_item: &ty::AssocItem) -> &'static str {
     match impl_item.kind {
         ty::AssocKind::Const => "const",
-        ty::AssocKind::Method => "method",
+        ty::AssocKind::Fn => "method",
```

Comment:
Another mention of "method", although this entire module might need a thorough de-method-izing.